whiteBoarding/squareDigits/keg-squareDigits.js

```diff
@@ -0,0 +1,8 @@
+function squareDigits(num) {
```

Comment:
Interesting solution and done without an array.  Is there another way to do this with higher order functions?

What are the pros and cons of using the higher order functions versus a simple for loop?

Reply:
It uses an array to store the results. You could do it all in line with toString().split('').map(with calculations in here).join('') probably. Not sure of advantages other than less code.

Reply:
That is an advantage, what is the disadvantage.  Write the code out in a similar function and run it.